libiocage/lib/Jail.py

```diff
@@ -601,7 +797,14 @@ def __getattr__(self, key):
 
         raise AttributeError(f"Jail property {key} not found")
 
-    def getattr_str(self, key):
+    def getstring(self, key):
```

Comment:
No it's not. When you don't intent to print the outputs, you don't want to get the strings. Some objects, like for example the JailConfigAddresses, are a custom data structure you want to have access to.